daml-foundations/daml-ghc/daml-prim-src/GHC/Generics.daml

```diff
@@ -0,0 +1,1685 @@
+{-# LANGUAGE DataKinds                  #-}
```

Comment:
Can we say at the top where we got it from, so we can track changes upstream easier.

Reply:
done.